recipes/roounfold/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "roounfold" %}
+{% set version = "3.0.5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.cern.ch/RooUnfold/RooUnfold/-/archive/{{ version }}/RooUnfold-{{ version }}.tar.gz
+  sha256: e57d846efcaff831dfeaad528fa9ebc60fe3e09165decbbc261592c23851d40a
+  patches:
+    - patches/cmake-minimum-3.5.patch
+
+build:
+  number: 0
+  script:
+    - cmake ${CMAKE_ARGS} -S . -B build
+    - cmake --build build --clean-first --parallel=${CPU_COUNT}
+    - cmake --install build
+    - export SP_DIR=$(python -c "import site; print(site.getsitepackages()[0])")  # [unix]
+    - set "SP_DIR=%PREFIX%\\Lib\\site-packages"  # [win]
+    - python -c "import shutil, os; shutil.copytree(os.path.join(os.environ['SRC_DIR'], 'python'), os.path.join(os.environ['SP_DIR'], 'RooUnfold'), dirs_exist_ok=True)"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cmake
+    - make
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - python  # [build_platform != target_platform]
+    - root_base  # [build_platform != target_platform]
+  host:
+    - python
+    - root_base
+  run:
+    - python
+    - root_base
```

Comment:
It is fine to have a `run` requirement lower bound.

Reply:
How does that work? How can I run with a ROOT version lower than what I need?

Reply:
`RooUnfold` doesn't need a lower bound I believe.